src/main/java/org/jabref/gui/search/GlobalSearchResultDialog.java

```diff
@@ -53,17 +52,19 @@ public GlobalSearchResultDialog(ExternalFileTypes externalFileTypes, UndoManager
 
     @FXML
     private void initialize() {
-        viewModel = new GlobalSearchResultDialogViewModel(stateManager, preferencesService);
+        viewModel = new GlobalSearchResultDialogViewModel(preferencesService);
 
         PreviewViewer previewViewer = new PreviewViewer(viewModel.getSearchDatabaseContext(), dialogService, stateManager);
         previewViewer.setTheme(preferencesService.getTheme());
         previewViewer.setLayout(preferencesService.getPreviewPreferences().getCurrentPreviewStyle());
 
-        FieldColumn fieldColumn = new FieldColumn(MainTableColumnModel.parse("field:library"));
+        //TODO: Create a special column
+        FieldColumn fieldColumn = new FieldColumn(MainTableColumnModel.parse("customLibrary"));
```

Comment:
Instead of creating a special MainTableColumn, I would just use a plain old TableColumn if you just bind a static value later, should work here too.